doc/UserGuide/BuildingRunningTesting/TestingUPP.rst

```diff
@@ -50,4 +65,4 @@ For repeated regression test runs, users can edit the ``rt.sh`` file and disable
 * ``w`` -- specify the work directory for test case job output
 * ``r`` -- specify the run directory containing baselines and ``.diff`` files for comparison of changes in results
 
-The following are legacy options for when ``rt.sh`` was not included within the UPP repository and may be ignored by developers: ``-b``, ``-u``, ``-c``, ``-t``.
\ No newline at end of file
+The following are legacy options from before ``rt.sh`` was included within the UPP repository; they may be ignored by developers: ``-b``, ``-u``, ``-c``, ``-t``.
```

Comment:
After line 66, you can add these two additional options: 

* ``d`` -- disable IFI tests even if IFI is available
* ``e`` -- do not build the UPP executable